app/models/tax_return.rb

```diff
@@ -44,7 +46,7 @@ class TaxReturn < ApplicationRecord
   validates :year, presence: true
 
   attr_accessor :status_last_changed_by
-  after_update :send_mixpanel_status_change_event
+  after_update :send_mixpanel_status_change_event, :record_internal_interaction
```

Comment:
Bumps up touch behavior on a tax_return to an "internal interaction" -- confirmed with product that we want all changes to a tax return to count as "actions" towards the interaction SLA metric.